duo-example-admin/pom.xml

```diff
@@ -63,8 +63,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>2.3.2</version>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>1.8</source>
```

Comment:
1.8 shouldn't be backwards incompatible, and bring the example to the same target as the client itself.

Reply:
lgtm!